### PR TITLE
[SD-3654] Add times to sluglines report

### PR DESCRIPTION
**Pull request by marwoodandrew**
Following feedback from Vince, times have been added and Domestic and Rest of World have been grouped seperatly

### CODE REVIEW

File: server/apps/desks.py [hunk 1]
```diff
@@ -279,43 +279,58 @@ class SluglineDesksResource(Resource):
 class SluglineDeskService(BaseService):
     SLUGLINE = 'slugline'
     OLD_SLUGLINES = 'old_sluglines'
+    VERSION_CREATED = 'versioncreated'
     HEADLINE = 'headline'
     NAME = 'name'
     PLACE = 'place'
+    GROUP = 'group'
```
Comment (sivakuna-aap) [inline server/apps/desks.py]: I suggest to move these constants except `GROUP` to `item.py`.

**Comment by sivakuna-aap** [comment]
:+1:

File: server/apps/desks.py (hunk 1 above)
Comment (marwoodandrew) [inline server/apps/desks.py]: core constants eh!

Comment (sivakuna-aap) [inline server/apps/desks.py]: yep :smiley:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**